src/initializers/kafka/base-kafka-handler.ts

```diff
@@ -73,10 +74,7 @@ abstract class Base<Input, Output> {
   }
 
   parseHeaders(headers: KafkajsType.IHeaders) {
-    return Object.keys(headers).reduce(
-      (m, k) => ({ ...m, [k]: headers[k].toString() }),
-      {}
-    );
+    return Object.keys(headers).reduce((m, k) => ({ ...m, [k]: headers[k].toString() }), {});
```

Comment:
nit: 
```suggestion
    return Object.entries(headers).reduce((m, [k, v]) => ({ ...m, [k]: v.toString() }), {});
```

Reply:
Actually I this is not supported in Node 10, so lets leave it as you did it

Reply:
This is prettier doing the auto-formatting. Didn't touch those lines. (I am wondering why prettier didn't work for you)